package-managers/nuget.md

```diff
@@ -1,25 +1,76 @@
 # NuGet \(.NET\)
-
 TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-nuget)
 
-## Connecting to a NuGet feed
 
+## Connecting to a NuGet feed
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
 
+### Find your feed URL
+
+#### Public feed
+There are two types of public feeds you can connect to. A public feed you own and one from the gallery. The only real difference here, is where to find the URL of the feed.

```

Comment:
I had never thought of it this way but you are right! There are two ways. Main focus should be on feeds you or your team own, though. Gallery is secondary (and good to have covered).

Plan is to have this "connect" dialog in the authenticated version one day, too.